File: QViz.NET/Periscope.QViz/JSON/TestCaseTag.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test Case Tag Details
	/// </summary>
	public class TestCaseTag
	{
		/// <summary>
		/// Identifier of the Test Case Tag Details
		/// </summary>
		public string testCaseTagId;

		/// <summary>
		/// Tag Object of the Test Case
		/// </summary>
		public Tag tag;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public TestCaseTag()
		{
			this.testCaseTagId = "";
			this.tag = new Tag();
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/TestCaseTag.java
package com.periscope.qviz.json;

/**
 * JSON Model for Test Case Tag Details
 */
public class TestCaseTag {

	/**
	 * Identifier of the Test Case Tag Details
	 */
	public String testCaseTagId;

	/**
	 * Tag Object of the Test Case
	 */
	public Tag tag;

	/**
	 * Default Constructor
	 */
	public TestCaseTag() {
		this.testCaseTagId = "";
		this.tag = new Tag();
	}

}
